lib/file_store/local_store.rb

```diff
@@ -68,7 +64,7 @@ def purge_tombstone(grace_period)
     end
 
     def get_path_for(type, upload_id, sha, extension)
-      "#{upload_path}/#{super(type, upload_id, sha, extension)}"
+      "/#{upload_path}/#{super(type, upload_id, sha, extension)}"
```

Comment:
Can you see if we can avoid all these `/` by joining paths the right way? :stuck_out_tongue_closed_eyes: